### DISCUSSION TITLE
Linking objects (props/items) seen in CreativeWorks

### DISCUSSION THREAD

**Discussion by gwire** [schemaorg/schemaorg#3294] [Q&A]
How would the relationship between an object (eg a "Product") and the work (such as a movie or TV episode) it is seen in?

This would probably be analogous to [character](https://schema.org/character), but be able to express known products (eg a specific phone or model of car seen in a movie) as well as specific fictional items (such as the [computer on Lost](https://lostpedia.fandom.com/wiki/Swan_computer)).

Is it correct/intended to use [mentions](https://schema.org/mentions) for things that are merely seen and not explicitly referred to?

**Comment by RichardWallis** [comment]
There is no specific 'appears in' property, about/subjectOf is probably too strong a relationship. So [mentions](https://schema.org/mentions) is the closest appropriate.
